lib/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -760,4 +760,80 @@ public static function get_style_variations() {
 		}
 		return $variations;
 	}
+
+
+	/**
+	 * Resolves relative paths in theme.json styles to theme absolute paths
+	 * and returns them in an array that can be embedded
+	 * as the value of `_link` object in REST API responses.
+	 *
+	 * @since 6.6.0
+	 *
+	 * @param WP_Theme_JSON_Gutenberg  $theme_json A theme json instance.
+	 * @return array An array of resolved paths.
+	 */
+	public static function get_resolved_theme_uris( $theme_json ) {
+		$resolved_theme_uris = array();
+
+		if ( ! $theme_json instanceof WP_Theme_JSON_Gutenberg || empty( $theme_json ) ) {
```

Comment:
The `empty` check is superfluous since if it were empty then it wouldn't be an instance of `WP_Theme_JSON_Gutenberg`.

Reply:
You got it 🦸🏻